### PR TITLE
[13.0] [FIX] mail_debrand: Fixed travis error.

### PR DESCRIPTION
**Pull request by Chandresh-SerpentCS**

### CODE REVIEW

File: mail_debrand/__manifest__.py [hunk 1]
```diff
@@ -14,6 +14,5 @@
     "license": "AGPL-3",
     "installable": True,
     "depends": ["mail"],
-    "development_status": "Stable",
```
Comment (pedrobaeza) [inline mail_debrand/__manifest__.py:17]: Please change it to `Production/Stable` instead

Comment (Chandresh-SerpentCS) [reply mail_debrand/__manifest__.py:17]: @pedrobaeza You mean `Production/Stable` would be a key?

Comment (pedrobaeza) [reply mail_debrand/__manifest__.py:17]: It would be the proper value for the key `development_status`.

Comment (Chandresh-SerpentCS) [reply mail_debrand/__manifest__.py:17]: Ok, I will modify the value.

**Review by bodedra — APPROVED** [review]
LG :+1:

**Comment by bodedra** [comment]
@pedrobaeza Can we fast-track this?

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**